utility/src/main/java/org/oppia/android/util/system/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+"""
+Utilities corresponding to simplify using Oppia Clock.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "oppia_clock",
+    srcs = [
+        "OppiaClock.kt",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+kt_android_library(
+    name = "oppia_clock_impl",
+    srcs = [
+        "OppiaClockImpl.kt",
+    ],
+    deps = [
+        ":oppia_clock",
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "oppia_clock_module",
```

Comment:
Let's explicitly define this as a prod module since we'll be having both prod & test modules throughout the codebase.

Reply:
Done